src/lv_settings/lv_settings.c

```diff
@@ -157,6 +159,17 @@ void lv_settings_set_group(lv_group_t * g)
     group = g;
 }
 
+/**
+ * Change the maximum width of settings dialog object
+ * @param settings pointer to settings object
+ * @param max_width maximum width of the settings container page
+ */
+void lv_settings_set_max_width(lv_obj_t * settings, lv_coord_t max_width)

```

Comment:
Is there any point pass `settings` if not used?

Reply:
I guess not, but thought it to include it so as to maintain consistency with the API.
I don't feel strongly about it either way. feel free to remove it should you choose to.